src/main/java/seedu/address/model/Model.java

```diff
@@ -74,6 +74,16 @@ public interface Model {
      */
     public List<Student> getClashingStudents(Student student);
 
+    /**
+     * Retrieves the list of students, adds up their paid amount, and returns the total paid amount
+     */
+    public double getTotalPaidAmount();
+
+    /**
+     * Retrieves the list of students, adds up their owed amonut, and returns the total owed amount
+     */
+    public double getTotalOwedAmount();
+
```

Comment:
`Comments should explain the WHAT and WHY aspects of the code, rather than the HOW aspect.`
You could consider changing the javadocs to follow this guideline in the code quality